<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>criminal_counterDockWidgetBase</class>
 <widget class="QDockWidget" name="criminal_counterDockWidgetBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>416</width>
    <height>701</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>criminal_counter</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt; font-weight:600;&quot;&gt;CRIMINAL COUNTER&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="margin">
       <number>15</number>
      </property>
      <property name="indent">
       <number>5</number>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QTabWidget" name="tab_Main">
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="tab_Case">
       <attribute name="title">
        <string>Case Input</string>
       </attribute>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>20</y>
          <width>321</width>
          <height>101</height>
         </rect>
        </property>
        <property name="title">
         <string>case list</string>
        </property>
        <widget class="QWidget" name="formLayoutWidget_2">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>30</y>
           <width>281</width>
           <height>53</height>
          </rect>
         </property>
         <layout class="QFormLayout" name="formLayout_2">
          <property name="horizontalSpacing">
           <number>15</number>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;select by rank:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;select by time:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="comboBox_Rank"/>
          </item>
          <item row="1" column="1">
           <widget class="QComboBox" name="comboBox_Time"/>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QGroupBox" name="groupBox_4">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>140</y>
          <width>321</width>
          <height>351</height>
         </rect>
        </property>
        <property name="title">
         <string>case info</string>
        </property>
        <widget class="QWidget" name="verticalLayoutWidget_2">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>30</y>
           <width>281</width>
           <height>301</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="sizeConstraint">
           <enum>QLayout::SetDefaultConstraint</enum>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QListWidget" name="list_case">
            <property name="whatsThis">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;case no.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="isWrapping" stdset="0">
             <bool>false</bool>
            </property>
            <property name="viewMode">
             <enum>QListView::ListMode</enum>
            </property>
            <property name="uniformItemSizes">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QGraphicsView" name="graphics_case"/>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QPushButton" name="button_run">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>490</y>
          <width>111</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Run Case</string>
        </property>
       </widget>
       <widget class="QPushButton" name="button_cancel">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>520</y>
          <width>111</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_Analysis_2">
       <attribute name="title">
        <string>Analysis</string>
       </attribute>
       <widget class="QPushButton" name="button_NodeSelect">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>20</y>
          <width>111</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Node Selection</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_10">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>20</y>
          <width>201</width>
          <height>32</height>
         </rect>
        </property>
        <property name="text">
         <string>Uses selected nodes for blocking the criminal</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QTableView" name="table_Node">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>60</y>
          <width>321</width>
          <height>81</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_11">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>200</y>
          <width>61</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Job List:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
       <widget class="QListView" name="list_PoliceJob">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>220</y>
          <width>321</width>
          <height>271</height>
         </rect>
        </property>
       </widget>
       <widget class="QWidget" name="horizontalLayoutWidget_2">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>150</y>
          <width>295</width>
          <height>32</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="button_add">
           <property name="toolTip">
            <string/>
           </property>
           <property name="whatsThis">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/plugins/CriminalCounter/add.png&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="styleSheet">
            <string notr="true">image: url(:/plugins/CriminalCounter/add.png);</string>
           </property>
           <property name="text">
            <string>+</string>
           </property>
           <property name="autoDefault">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="button_substract">
           <property name="toolTip">
            <string/>
           </property>
           <property name="whatsThis">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/plugins/CriminalCounter/add.png&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="styleSheet">
            <string notr="true">image: url(:/plugins/CriminalCounter/add.png);</string>
           </property>
           <property name="text">
            <string>-</string>
           </property>
           <property name="autoDefault">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="button_calculate">
           <property name="text">
            <string>Calculation</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="layoutWidget_2">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>500</y>
          <width>321</width>
          <height>66</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="1" column="0">
          <widget class="QPushButton" name="button_StartCounter">
           <property name="text">
            <string>Start Counter</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QPushButton" name="button_undo">
           <property name="text">
            <string>Undo</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0" colspan="2">
          <widget class="QProgressBar" name="progress_job">
           <property name="value">
            <number>24</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_Reporting_2">
       <attribute name="title">
        <string>Reporting</string>
       </attribute>
       <widget class="QLabel" name="label_12">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>20</y>
          <width>141</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Action Summary:</string>
        </property>
       </widget>
       <widget class="QListView" name="list_summary">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>40</y>
          <width>321</width>
          <height>231</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_13">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>290</y>
          <width>111</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Graphic Report:</string>
        </property>
       </widget>
       <widget class="QGraphicsView" name="graphics_report">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>310</y>
          <width>321</width>
          <height>201</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="button_clear">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>650</y>
          <width>111</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Clear Report</string>
        </property>
       </widget>
       <widget class="QPushButton" name="button_save">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>520</y>
          <width>111</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Save Report</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
